imageio/plugins/ffmpeg.py

```diff
@@ -512,7 +514,8 @@ def _load_infos(self):
             if match is not None:
                 hms = map(float, line[match.start()+1:match.end()].split(':'))
                 self._meta['duration'] = duration = cvsecs(*hms)
-                self._meta['nframes'] = int(round(duration*fps))
+                if fps:
+                    self._meta['nframes'] = int(round(duration*fps))
```

Comment:
@almarklein the way I got the frame count in final cut pro was to actually add the video to a project timeline and then look at the timecode in frames. That way I knew for sure of the total frame count since that forces the whole file to be read in. So I'm essentially reading all the frames.

I don't know if we would want to get the true frame count by seeiking to the end of a file because you would have to read in the whole file just to get the frame count. That seems like a potentially heavy computation if you have a big video file (which could be gigabytes to read in). I think it's best just to get the estimate frame count and deal with it if its not exactly correct. At least that's my opinion.